gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-6/api-creation-v4-step-6.component.ts

```diff
@@ -86,9 +86,8 @@ export class ApiCreationV4Step6Component implements OnInit {
     // TODO: send info to correct endpoint to create and publish the new API
   }
 
-  onChangeStepInfo(stepNumber: number) {
-    // Change current step to step number
-    this.stepService.goToStepPosition(stepNumber);
+  onChangeStepInfo(stepLabel: string) {
```

Comment:
Passing the label manually in the html could cause problems in the future if the text ever changes. Since each of the steps in the summary are 'primary' steps, could you create a method that navigates directly to the primary of a given step?

Reply:
I have change because I preferred to switch to the business labels. The number 1 2 3 it's not very practical because we don't know what it corresponds to. And if the order changes, we should also think about changing here. 
but we can add a uniqueness constraint on the steps, wdyt ?